charts/gardener/provider-local/internal/shoot-system-components/charts/local-path-provisioner/templates/deployment.yaml

```diff
@@ -19,6 +19,7 @@ spec:
       labels:
         app: local-path-provisioner
     spec:
+      priorityClassName: gardener-system-200
```

Comment:
How does this work? This component is running in the shoot and here we should only see `gardener-shoot-system-*` priority classes. 
In this light, `system-cluster-critical` is more appropriate (CSI drivers are using `system-node-critical` but those are `DaemonSet`s)